src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -2066,6 +2086,46 @@ abstract class DatabaseTest {
     cohortModulesDao.insert(row)
   }
 
+  fun insertEvent(
+      id: Any? = null,
+      moduleId: Any = inserted.moduleId,
+      eventType: EventType = EventType.Workshop,
+      meetingUrl: URI? = null,
+      slidesUrl: URI? = null,
+      recordingUrl: URI? = null,
```

Comment:
Might be convenient to make these `Any?` and then do `meetingUrl?.let { URI("$it") }` so callers can pass these as strings.